public/components/health-check/container/health-check.container.tsx

```diff
@@ -67,39 +65,26 @@ const checks = {
     awaitFor: ["api"],
   },
   pattern: {
-    title: 'Check alerts index pattern',
-    label: 'Alerts index pattern',
+    title: 'Check index pattern',
+    label: 'Index pattern',
     validator: checkPatternService,
     awaitFor: [],
-    canRetry: true,
-  },
-  template: {
-    title: 'Check alerts indices template',
-    label: 'Alerts indices template',
-    validator: checkTemplateService,
-    awaitFor: ["pattern"],
-    canRetry: true,
-  },
-  fields: {
-    title: 'Check alerts index pattern fields',
-    label: 'alerts index pattern fields',
-    validator: checkFieldsService,
-    awaitFor: ["pattern"],
+    shouldCheck: true,
     canRetry: true,
   },
   patternMonitoring: {
     title: 'Check monitoring index pattern',
     label: 'Monitoring index pattern',
     validator: (appConfig) => checkPatternSupportService(appConfig.data['wazuh.monitoring.pattern'], WAZUH_INDEX_TYPE_MONITORING),
-    awaitFor: [],
+    awaitFor: ["pattern"],
     shouldCheck: true,
     canRetry: true,
   },
   patternStatistics: {
     title: 'Check statistics index pattern',
     label: 'Statistics index pattern',
     validator: (appConfig) => checkPatternSupportService(`${appConfig.data['cron.prefix']}-${appConfig.data['cron.statistics.index.name']}-*`, WAZUH_INDEX_TYPE_STATISTICS),
-    awaitFor: [],
+    awaitFor: ["pattern"],
```

Comment:
Why this check is waiting for the check of alert index pattern? These are not related.

Reply:
Removed in last commit